app/views/components/gds/saveAsDraft.scala.html

```diff
@@ -0,0 +1,21 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.util.SaveAndReturn
+
+@(messageKey: String = "site.save_and_come_back_later")(implicit messages: Messages)
+<br>
```

Comment:
I am not sure that is good to have structure elements in component.
It is break hermetization from component perspective.

Reply:
This component is a bit of a hack to get around the fact that the gds prototyping kit doesn't appear to have a way of rendering a submit button as a link.

Maybe I should render both the "save and continue" and "save as draft" buttons in this component (as they are nearly always used together anyway).    In that case, would the <br> be appropriate?

Reply:
Maybe be having universal `save and continue` and build over it second component `declaration_page_actions` or something like this? We have two birds at once
* we have components that have clear boundaries and could be used outside each other
* we can be sure that each page that have save, have also continue

On the other hand is this should not be secondary button ?